pontoon/tour/migrations/0002_make_tutorial_public.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.13 on 2020-10-10 22:57
+
+from django.db import migrations
+
+
+def make_tutorial_public(apps, schema_editor):
+    """Tutorial should be public by default."""
+
+    Project = apps.get_model("base", "Project")
+    Project.objects.filter(slug="tutorial").update(visibility="public")
```

Comment:
We've already made all projects public in https://github.com/mozilla/pontoon/blob/master/pontoon/base/migrations/0004_make_old_projects_public_again.py#L10.

Why are these migrations needed?

Reply:
https://github.com/mozilla/pontoon/pull/1682#discussion_r502843775

Reply:
Ah, missed first part of your comment.

Now I see where the problem see. I'll leave it to you to decide what's best to do here - update the migration or add a new one.